examples/02-HFSS/Probe_Fed_Patch.py

```diff
@@ -0,0 +1,113 @@
+"""
+HFSS: Probe-fed patch antenna
+---------------------------------------------------------
+This example shows how to use the ``Stackup3D`` class
+to create and analyze a patch antenna in HFSS 3D.
+
+Note that the HFSS 3D Layout interface may offer advantages for
+laminate structures such as the patch antenna.
+"""
+
+###########################
+# Perform  imports
+# ~~~~~~~~~~~~~~~~~~
+
+import os
+
+import pyaedt
+import tempfile
+from pyaedt.modeler.advanced_cad.stackup_3d import Stackup3D
+
+###############################################################################
+# Set non-graphical mode
+# ~~~~~~~~~~~~~~~~~~~~~~
+# Set non-graphical mode. ``"PYAEDT_NON_GRAPHICAL"`` is set to ``False``
+# to create this documentation.
+#
+# You can set ``non_graphical``  to ``True`` to view
+# HFSS while the notebook cells are executed.
+#
+# Use the 2023R2 release of HFSS.
+
+non_graphical = False  # Set to False to launch the AEDT UI.
+desktop_version = "2023.2"
+length_units = "mm"
+freq_units = "GHz"
+
+########################################################
+# Create temporary working folder
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Use tempfile to create a temporary working folder. Project data
+# is deleted after this example is run.
+#
+# To save the project data in another location, change
+# the location of the project directory.
+#
+
+# tmpdir.cleanup() at the end of this notebook removes all
+# project files and data.
+
+tmpdir = tempfile.TemporaryDirectory(suffix="_aedt")
```

Comment:
@dcrawforAtAnsys for me it's fine to use tempfile but we could also rely on the shutil functionalities inside the Scratch class in filesystem.py if needed.
In this specific case you just need to create a temp folder and you can do so just by using pyaedt.generate_unique_folder_name() and then use remove method in Scratch class.